explainer/explainer11.md

```diff
@@ -1,49 +1,52 @@
 
-The WoT Architecture document serves as an umbrella for all WoT specifications, and hence also sets the general goals of the W3C Web of Things.
-It defines the abstract architecture and spans the design space for current and future so-called building blocks, which are specified in separate documents.
-Specifically, the purpose of the WoT Architecture document is to provide
+The WoT Architecture document serves as an umbrella for all WoT specifications, and sets the general goals of the W3C Web of Things.
+It defines the abstract architecture and the design space for current and future building blocks, each of which are specified in separate documents.
+Specifically, the purpose of the WoT Architecture document is to provide:
+- basic terminology definitions;
+- use cases and derived requirements the W3C WoT Architecture should satisfy;
+- definitions of basic concepts and architectural constraints;
+- an overview of the WoT building blocks and their interplay; and
+- a guide for mapping the abstract architecture onto a variety of concrete deployment scenarios.
 
-- the basic terminology,
-- use cases and derived requirements the W3C WoT Architecture should satisfy,
-- definitions of the basic concepts and architectural constraints,
-- an overview over the WoT building block documents and their interplay, and
-- a guide for mapping the abstract architecture onto a variety of concrete deployment scenarios
-
-The architecture document lays the foundation for the WoT building block specifications:
+The architecture document lays the foundation for the following WoT building block specifications:
 
 - the <a href="https://w3c.github.io/wot-thing-description/">WoT Thing Description</a> (REC Track),
 - the <a href="https://w3c.github.io/wot-profile/">WoT Profile</a>,
 - the <a href="https://w3c.github.io/wot-binding-templates/">WoT Binding Templates</a>,
 - the <a href="https://w3c.github.io/wot-discovery/">WoT Discovery</a>, and
-- the <a href="https://w3c.github.io/wot-security/">WoT Security and Privacy Considerations</a>.
+- the <a href="https://w3c.github.io/wot-security/">WoT Security and Privacy Guidelines</a>.
 
-Of these, the WoT Security and Privacy Considerations provide a general analysis that cross-cuts the other building blocks, each of which also include their own specific security and privacy considerations.
+Of these, the WoT Security and Privacy Guidelines provide a general analysis that cross-cuts the other building blocks, each of which also include their own specific security and privacy considerations.
 
-## Goals
+In addition, the following optional/informative building blocks are defined:
```

Comment:
I would remove "optional". Maybe use "...the following complementary building blocks are defined:"

Reply:
Suggest to reword to "the following informative building blocks are defined, which complement the core of the Web of Things"

Reply:
Adopted Michael Lagally's suggested wording.